tests/integration/tests/conftest.py

```diff
@@ -86,9 +88,32 @@ def instances(
         instances.append(instance)
         util.setup_k8s_snap(instance, snap_path)
 
-    first_node, *_ = instances
-    first_node.exec(["k8s", "bootstrap"])
+    # first_node, *_ = instances
+    # first_node.exec(["k8s", "bootstrap"])
 
     yield instances
 
     _harness_clean(h)
+
+
+@pytest.fixture(scope="function")
+def etcd_count(request) -> int:
+    etcd_count_marker = request.node.get_closest_marker("etcd_count")
+    if not etcd_count_marker:
+        return 0
+    etcd_count_arg, *_ = etcd_count_marker.args
+    return int(etcd_count_arg)
+
+
+@pytest.fixture(scope="function")
+def etcd_cluster(
+    h: harness.Harness, etcd_count: int
+) -> Generator[List[harness.Instance], None, None]:
```

Comment:
This yields an etcd cluster object right?

```suggestion
) -> Generator[EtcdCluster, None, None]:
```